misc/python/materialize/mzcompose/composition.py

```diff
@@ -374,10 +390,7 @@ def invoke(
                         stdout=e.stdout,
                         stderr=e.stderr,
```

Comment:
Don't you need it here as well?

Reply:
Need what? This looks correct to me, we pass stdout and stderr to the exception so the caller has all the context of what failed.

Reply:
I thought you would need to use your collected `stdout_result` and `stderr_result` here as well. This is not the case?

Reply:
No, because the exception e contains the fields stdout and stderr, which we previously fill with stdout_result and stderr_result respectively in line 361

Reply:
Alright!